### PR TITLE
Revert "Revert "add instructions for deploy to wordpress""

### PR DESCRIPTION
**Pull request by RobertGardner**
Could you look at my questions in the README? Thanks.

### CODE REVIEW

File: deploy/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Deploying the search client to a custom Wordpress page template
+
+1. Run `npm run build` to build this project.
+2. Figure out which Wordpress theme you're using and navigate to the directory for that theme under wp\_content.
```
Comment (RobertGardner) [inline deploy/README.md:4]: It took me awhile to figure out, but I think the README has escaped `_` for some reason. Could you confirm so I can remove the `\`s?

Comment (RobertGardner) [inline deploy/README.md:4]: Is `wp_content` a directory in Wordpress?

File: deploy/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+3. Add the code in ourroots\_cms\_functions.php to the end of the functions.php file in that directory. If you are upgrading a previous installation, you may want to change the version numbers from 0.0.1 to something else for cache-busting.
```
Comment (RobertGardner) [inline deploy/README.md:5]: 'that directory' refers to the one in Wordpress under `wp_content`?

File: deploy/README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+4. Copy ourroots\_cms.php, main.css, and main.js from this directory to that directory.
+5. Copy the .js files from build/static/js to that directory, renaming main....chunk.js to main.chunk.js and renaming the other ...chunk.js file to other.chunk.js.
```
Comment (RobertGardner) [inline deploy/README.md:7]: I don't know how often we're going to do this, or how hard it is to rename, but if we do it frequently (or automated), could we write a simple script to do this rename?

For that matter, can we write a script to do the whole thing?

File: deploy/README.md (hunk 1 above)
Comment (DallanQ) [reply deploy/README.md:4]: I thought that _'s needed to be escaped in markdown files, but they don't need to be escaped. I removed the \'s.

Comment (DallanQ) [reply deploy/README.md:4]: it should be wp-content, but yes it is.

File: deploy/README.md (hunk 2 above)
Comment (DallanQ) [reply deploy/README.md:5]: yes, I clarified this.

File: deploy/README.md (hunk 3 above)
Comment (DallanQ) [reply deploy/README.md:7]: Most people use filezilla or some other ftp client to manage their wordpress installations. I'm not sure a script would be that helpful.

**Comment by DallanQ** [comment]
Updated the readme to clarify things and correct an omission to create an ourroots_cms subdirectory.

File: deploy/README.md (hunk 3 above)
Comment (RobertGardner) [reply deploy/README.md]: Got it. I thought these were instructions for us, the developers, not for end users. Looks good.

**Pull request merged**